src/main/java/mekanism/api/transmitters/IGridTransmitter.java

```diff
@@ -64,4 +64,6 @@
     void updateShare();
 
     Object getBuffer();
+
+    boolean isCompatibleWith(IGridTransmitter<A,N> other);
```

Comment:
We should probably bump the api version in https://github.com/mekanism-mod/Mekanism/blob/1.12/src/main/java/mekanism/api/transmitters/package-info.java to at least 9.1 or something given this is a change in API.

Reply:
agreed

Reply:
I can add a default to the interface too

Reply:
Sounds good, not aware of anyone else using this part of our API but no reason to break things on people when we don't have to.